Ginger/GingerCore/Repository/RepositorySerializer.cs

```diff
@@ -489,7 +489,8 @@ private static object xmlReadObject(Object Parent, XmlReader xdr, RepositoryItem
 
                 if (obj == null)
                 {
-                    throw new Exception("Error:Cannot create Class: " + ClassName);
+                      Reporter.ToLog(eLogLevel.INFO, "Error:Cannot create Class: " + ClassName, writeOnlyInDebugMode: true);
+                      return null;
```

Comment:
issue with this is that we might miss that there is some issue with some class, at least change the Log to be: eLogLevel.Error and remove this writeOnlyInDebugMode: true- it always needs to be added to Log

Reply:
changed the Log from info to error and removed writeOnlyInDebugMode: true.